server/cli/launch.py

```diff
@@ -54,6 +55,8 @@
 @click.option("--obs-names", default=None, metavar="", help="Name of annotation field to use for observations.")
 @click.option("--var-names", default=None, metavar="", help="Name of annotation to use for variables.")
 @click.option("--host", default="127.0.0.1", help="Host IP address")
+@click.option("--fixed-port", is_flag=True, default=False, show_default=True,
+              help="Error if request port is not available instead of searching for another available port.")
```

Comment:
I'm not sure the help message is clear.  Ideas:
* rename option to `-no-port-scan`
* help="Disable automatic scan for unused port"

Reply:
+1

Reply:
For what it's worth, I think it might be easier to just have `--port`, and if it's not specified, just go and find a port.

Reply:
yes, much better. naming things is hard 😞